robot_smach_states/src/robot_smach_states/navigation/guidance.py

```diff
@@ -338,28 +346,22 @@ def _reset_mentioned_entities(userdata=None):
                 Resets the entities that have been mentioned so that the robot will mention all entities to all
                  operators
                  """
+                self.robot.head.look_at_point(VectorStamped.from_xyz(-1.0, -0.1, 1.75, stamp=rospy.Time.now(),
+                                                                     frame_id=self.robot.base_link_frame))
+                rospy.sleep(3.0)
```

Comment:
I think it would be better to add a second callback state to move the head to keep these functions separate from the internal resetting of the tourguide